doc/dftb+/manual/dftbp.tex

```diff
@@ -4197,6 +4198,9 @@ \section{Analysis}
   charge associated only with the on-site part of the atomic orbitals, not the
   off-site hybridisation).
 
+\item[\is{WriteCosmoFile}] If \is{Yes}, the cavitity information is written as
+  cosmo file to dftbp.cosmo. Only works if the solvation model is COSMO.
```

Comment:
```suggestion
  a cosmo file to dftbp.cosmo. Only works if the solvation model is COSMO.
```
Is there an available reference for what is in a cosmo file (I'm guessing this is for COSMOtherm/COSMO-RS or similar)?

Reply:
Good question, we tried to reconstruct them from the Turbomole generated cosmo files. Of course this file format is far from being standardized in any way. Other implementations outside of the Turbomole package are rare, not sure if they use COSMO files:
- https://github.com/usnistgov/COSMOSAC
- https://www.scm.com/doc/COSMO-RS